src/common/com/intellij/plugins/haxe/ide/generation/GetterSetterMethodBuilder.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2000-2013 JetBrains s.r.o.
+ * Copyright 2014-2017 AS3Boyan
+ * Copyright 2014-2014 Elias Ku
+ * Copyright Fedor Korotkov
```

Comment:
Pull Fedor's name.

Fedor's contributions are under JetBrain's copyright. He was working for them at the time and hasn't touched the plugin since then. So this message is inappropriate. Despite your good intentions, you should never include any copyright message beside your own.